Source/GUI/MemWatcher/Dialogs/DlgAddWatchEntry.cpp

```diff
@@ -424,3 +437,23 @@ MemWatchEntry* DlgAddWatchEntry::stealEntry()
   m_entry = nullptr;
   return entry;
 }
+
+void DlgAddWatchEntry::onPointerOffsetContextMenuRequested(const QPoint& pos)
+{
+  QLabel* const lbl = static_cast<QLabel*>(sender());
+
+  QMenu* contextMenu = new QMenu(this);
```

Comment:
Nit: For correctness, both `contextMenu` (and `copyAddr` below) could be marked as `const`, so that the pointers cannot be reassigned inadvertently.